components/Assets/AssetListDisplay.tsx

```diff
@@ -0,0 +1,31 @@
+import { Feather } from '@expo/vector-icons';
+import { View } from 'react-native';
+import { Token } from 'types/supabaseTypes';
+
+import { FText } from '../Text/FText';
+
+interface AssetListDisplayProps {
+  token: Token;
+}
+
+export const AssetListDisplay = ({ token }: AssetListDisplayProps) => {
+  // const actualValue = token.value[-1].value;
```

Comment:
Is this for testing or can we remove it?

Reply:
This is until Kentin fixes the issue with the data format consistency, until then we are using mock data